Bench-Kitura-Blog/build.sh

```diff
@@ -31,12 +31,33 @@ Darwin)
   ;;
 esac
 
-# Build 'new' version
+# Build 'new' and "nio" version
 # If $REPO and $NEW_COMMIT are set, edit package $REPO to $NEW_COMMIT before building
+
+function cloneProject {
```

Comment:
This should probably go into `Bench-Swift/lib/build.sh` so that you don't have to duplicate it in every test suite.

Reply:
Have made the suggested changes.